src/rpc/blockchain.cpp

```diff
@@ -694,6 +695,7 @@ static UniValue getblockheader(const JSONRPCRequest& request)
             "  \"bits\" : \"1d00ffff\", (string) The bits\n"
             "  \"difficulty\" : x.xxx,  (numeric) The difficulty\n"
             "  \"chainwork\" : \"0000...1f3\"     (string) Expected number of hashes required to produce the current chain (in hex)\n"
+            "  \"nTx\" : n,             (numeric) The number of transactions in the block. This result can also be returned for pruned blocks, as it's stored in the block index.\n"
```

Comment:
> This result can also be returned for pruned blocks, as it's stored in the block index

Should be a comment in the source code instead?

I believe we have a pruning test, so it would make sense to add an `assert_equal(nTx, whatever)` there?